src/IotjsDebugger.ts

```diff
@@ -109,13 +112,79 @@ class IotjsDebugSession extends DebugSession {
       return;
     }
 
-    this._args = args;
+    this._attachArgs = args;
     if (args.debugLog in LOG_LEVEL) {
       this._debugLog = args.debugLog;
     } else {
       this.sendErrorResponse(response, new Error('No log level given'));
     }
 
+    this.connectToDebugServer(response, args);
+  }
+
+  protected async launchRequest(response: DebugProtocol.LaunchResponse, args: ILaunchRequestArguments) {
+
+    if (!args.address || args.address === '') {
+      this.sendErrorResponse(response, new Error('Must specify an address'));
+      return;
+    }
+    if (!args.localRoot || args.localRoot === '') {
+      this.sendErrorResponse(response, new Error('Must specify a localRoot'));
+      return;
+    }
+
+    this._launchArgs = args;
+    if (args.debugLog in LOG_LEVEL) {
+      this._debugLog = args.debugLog;
```

Comment:
You've checked all used args parameter that it exists, except this `debugLog`. Why?
It will also lead to error when removed from the json file. 
Please check it also, or we could suppose that they (localRoot, adress, etc.) are exist, which would ease the checks above. @rtakacs @tdusnoki : What is your opinion?

Reply:
So, the undefined args.debugLog does not leads to ReferenceError?